### PR TITLE
css/tagsinput-switch

### LINKED ISSUE #1895: convert css for tagsinput

### LINKED ISSUE #1896: convert css for tag

### LINKED ISSUE #1897: convert css for table

### LINKED ISSUE #1898: convert css for tab

### LINKED ISSUE #1899: convert css for switch

### PR DESCRIPTION
**Pull request by EdwardIrby**
resolves #1895
resolves #1896
resolves #1897
resolves #1898 
resolves #1899

### CODE REVIEW

**Comment by jaketrent** [comment]
I'll take a look at this. Why are there old, unrelated commits in this new branch?

File: packages/tab/src/react/list-item.tsx [hunk 1]
```diff
@@ -57,17 +35,21 @@ const ListItem = React.forwardRef<ListItemElement, ListItemProps>(
       'href' in props ? 'a' : 'button',
       {
         ...rest,
-        ...styles.listItem(active || false, themeName),
+        className: classNames(
```
Comment (jaketrent) [inline packages/tab/src/react/list-item.tsx:38]: This is an exported sub-component. Please add className composition.

File: packages/tab/src/react/list.tsx [hunk 2]
```diff
@@ -192,20 +166,28 @@ const List: React.FC<React.HTMLAttributes<HTMLDivElement>> = props => {
...
     role: 'tablist',
     onKeyDown: handleKeyDown,
     tabIndex: 0
   }
   return (
-    <div {...listProps} {...styles.list(themeName)} ref={listRef}>
+    <div
+      {...listProps}
+      className={classNames(
+        className,
```
Comment (jaketrent) [inline packages/tab/src/react/list.tsx:180]: I would compose last.

File: packages/table/src/css/index.css [hunk 3]
```diff
@@ -0,0 +1,186 @@
+.psds-table__container,
+null.psds-table__container {
```
Comment (jaketrent) [inline packages/table/src/css/index.css:2]: please cleanup.

File: packages/table/src/css/index.css [hunk 4]
```diff
@@ -0,0 +1,186 @@
...
+  cursor: pointer;
+}
+.psds-table__header--sortable.psds-theme--dark:hover {
+  color: var(--ps-colors-text-icon-high-on-dark);
+}
+.psds-table__header--sortable.psds-theme--light:hover {
+  color: var(--ps-colors-text-icon-high-on-dark);
+}
+.psds-table__header__sort-icon.psds-table__header__sort-icon,
+null.psds-table__header__sort-icon.psds-table__header__sort-icon {
```
Comment (jaketrent) [inline packages/table/src/css/index.css:97]: cleanup and dedup

File: packages/table/src/react/index.tsx [hunk 5]
```diff
@@ -121,10 +45,20 @@ const Table = React.forwardRef<HTMLTableElement, TableProps>((props, ref) => {
   return (
     <Container
       ref={containerRef}
-      {...styles.container(themeName, { scrollable })}
+      className={classNames(
+        className,
```
Comment (jaketrent) [inline packages/table/src/react/index.tsx:49]: would compose last.

there are a few more, so I'm going to stop marking these.

File: packages/tag/src/css/index.css [hunk 6]
```diff
@@ -0,0 +1,63 @@
...
+.psds-tag--size-medium {
+  height: 32px;
+  padding: 0 var(--ps-layout-spacing-medium);
+  font-size: var(--ps-type-font-size-200);
+  line-height: var(--ps-type-line-height-extra);
+}
+.psds-tag--icon {
+  padding-right: var(--ps-layout-spacing-xxsmall);
+}
+.psds-tag--isPressed {
```
Comment (jaketrent) [inline packages/tag/src/css/index.css:45]: dashify

File: packages/tagsinput/src/css/index.css [hunk 7]
```diff
@@ -0,0 +1,55 @@
+.psds-tagsinput,
+null.psds-tagsinput {
```
Comment (jaketrent) [inline packages/tagsinput/src/css/index.css:2]: cleanup

File: packages/tagsinput/src/css/index.css [hunk 8]
```diff
@@ -0,0 +1,55 @@
...
+  min-height: 40px;
+}
+.psds-tagsinput--disabled {
+  pointer-events: none;
+}
+.psds-tagsinput__prefix {
+  align-items: center;
+  display: -webkit-box;
```
Comment (jaketrent) [inline packages/tagsinput/src/css/index.css:10]: vendor-specific not needed

File: packages/text/src/css/index.ts [hunk 9]
```diff
@@ -0,0 +1,207 @@
+import {
```
Comment (jaketrent) [inline packages/text/src/css/index.ts:1]: this seems like a mistake to be re-appearing.

**Review by jaketrent — CHANGES_REQUESTED** [review]

**Review by sweeting — CHANGES_REQUESTED** [review]
@EdwardIrby  

Did a visual check on:
✅ switch
✅ tab
✅ table
✅ tag
⛔ tagsinput

tagsinput: looks like the text & icon color in the tags is getting turned dark somewhere.
CleanShot 2021-07-07 at 13 52 27

File: packages/switch/src/react/index.tsx [hunk 10]
```diff
@@ -161,14 +93,32 @@ const Switch = React.forwardRef<HTMLInputElement, SwitchProps>(
 
     return (
       <label
-        className={className}
+        className={classNames(
+          className,
```
[the remark this answers is not in the document]
Comment (EdwardIrby) [reply packages/switch/src/react/index.tsx:97]: The order in which the attributes are overwritten is not determined by the order the classes are defined in the class attribute, but instead where they appear in the CSS.

**Review by sweeting — CHANGES_REQUESTED** [review]
tagsinput: Still seeing the dark text & icons in the blue tags.

**Review by sweeting — APPROVED** [review]
👍 
CleanShot 2021-07-08 at 12 21 40

**Review by jaketrent — APPROVED** [review]

**Pull request merged**